docs/Product-Overview.md

```diff
@@ -0,0 +1,72 @@
+# Product Overview
+
+## OpenTripPlanner project
+
+OpenTripPlanner is a group of open source software applications that help individuals and organizations 
+identify and digitize multimodal trip plans based on OpenStreetMap (OSM) and other standardized data 
+sources (e.g. GTFS, GBFS, NeTEx).
+
+OpenTripPlanner is a community of dozens of individuals and organizations that work collaboratively to 
+improve multimodal trip planning best practices and to make it easier for public transit agencies and 
+public transit riders to publish and access information about transit services.
+
+OpenTripPlanner is locally managed in as many different ways as there are OpenTripPlanners projects. 
+OpenTripPlanner consistently and dependably delivers multimodal trip plans to millions or riders 
+everyday in dozens of countries around the globe. The project is actively maintained by the community, 
+with more than 50 commits most weeks during 2022, and 20 different developers having made 50 or more 
+commits during the life of the project.
+
+## What exactly is OTP?
+
+The most commonly used OpenTripPlanner component is a server-side Java application that is primarily 
+designed to import map, transit service, and other mobility data sets, input user origins and 
+destinations, and output API endpoints that allow other applications to consume generated trip plans, 
+vector map tiles, stop departures, and other rider information.
+
+In other words, OTP is a backend application that can work with other frontend user interfaces. OTP 
+works with your website, mobile app, physical signage or other applications in order to provide 
+relevant customer information and a meaningful trip plan to riders.
+
+There's no official OTP user interface, although the OpenTripPlanner ecosystem includes several user 
+interface projects. How your OTP looks is entirely up to you.
```

Comment:
Again, it might be better to refer to an OTP-based system or OTP deployment rather than "your OTP" lest people get the impression that having a custom application built on OTP invoves forking the OTP component (rather than just deploying it and layering / interconnecting other components with it).

Reply:
added deployment